src/core/styles/_constants.scss

```diff
@@ -20,7 +20,10 @@ $font-weight-bold: 700;
 
 // Primary colors
 $color-black: hsl(0deg 0% 0%); // #000
+
+// White colors
 $color-white: hsl(0deg 0% 100%); // #fff
+$color-white-800: hsl(0deg 0% 100% / 80%); // #fff
```

Comment:
There is no need to introduce a new color, you could you `background-color: rgba($color-white, $opacity-80);` for that purpose.

Also,
```
$color-white: hsl(0deg 0% 100%); // #ffffff
$color-white-800: hsl(0deg 0% 100% / 80%); // #ffffffcc
```
And the naming is not the best, since $color-white-800 suggests a darker shade of white, but it actually represents a semi-transparent version of white

Reply:
Thanks. Done